src/ngraph/op/util/logical_reduction.hpp

```diff
@@ -29,6 +29,13 @@ namespace ngraph
             class LogicalReduction : public Op
             {
             public:
+                /// \brief Constructs a logical reduction operation.
+                LogicalReduction();
```

Comment:
Probably want to change the ctors here to `protected`, like you did in `BinaryElementwiseArithmetic`.